pibic-unioeste.csl

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>PIBIC-Unioeste</title>
+    <title-short>PIBIC</title-short>
+    <id>http://www.zotero.org/styles/pibic-unioeste</id>
+    <link href="http://www.zotero.org/styles/pibic-unioeste" rel="self"/>
+    <link href="http://www.zotero.org/styles/pibic-unioeste" rel="template"/>
+    <link href="labceuunioeste.wix.com/unioeste" rel="documentation"/>
```

Comment:
Just in case it isn't clear, I'm asking whether there a description exists of the citation format (what the references should look like). I'm not asking for a description of the CSL style itself.